tracing/README.md

```diff
@@ -0,0 +1,188 @@
+Distributed Tracing
+---
+
+Both Helidon SE and Helidon MP use Opentracing API for tracing
+events.
+
+_Eclipse Microprofile Opentracing specification is not yet implemented_
+
+#Usage
+## Usage In Helidon MP
+To use tracing in MP, simply depend on the `helidon-microprofile-tracing` module to automate
```

Comment:
"simply depend on" --> I suggest restating this more explicitly; i.e., "...add the <blah> module as a dependency in the POM..."